jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/SystemConfig.java

```diff
@@ -0,0 +1,37 @@
+package us.dot.its.jpo.ode;
+
+public class SystemConfig implements SystemConfigMBean {

```

Comment:
I don't see the point of this class. Looks like it was written for a tutorial and NOT relevant to our applications.